.travis.yml

```diff
@@ -22,7 +22,11 @@ addons:
 matrix:
   include:
     - os: linux
-      compiler: clang
+      compiler: gcc
+      env: BUILD_PYTHON=ON
```

Comment:
We could use both 'gcc' and 'clang' with two specific env ?

Reply:
Ok, whatever